### ISSUE TITLE
Add playbook duration in logstash callback stats

### ISSUE THREAD

**Issue by rrey** [ansible/ansible#25781]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
lib/ansible/plugins/callback/logstash.py

##### ANSIBLE VERSION

```
ansible 2.3.0.0
  config_file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.9 (default, Jun 29 2016, 13:08:31) [GCC 4.9.2]
```

##### CONFIGURATION

Simply enabled logstash callback:
```
callback_whitelist = logstash
```

##### SUMMARY
The  event sent to logstash does not contain the playbook runtime statistic.
The information seems pretty useful and should be added to allow fast statistic creation on playbook execution time.

##### EXPECTED RESULTS
A field in the logstash event that contains the paybook runtime duration.

##### ACTUAL RESULTS
We currently only have the timestamp associated to the ansible_type "finish". One would have to retrieve the timestamp of the playbook start time and compute the duration to be able to do stats on playbook duration.

**Comment by jctanner** [comment]
cc @ujenmr

**Comment by rrey** [comment]
I made a pull request implementing this feature : #25785

**Comment by ujenmr** [comment]
I implemented this feature in #20725

**Comment by rrey** [comment]
Hi @ujenmr, my PR was merged in August 2017 so the feature is already in ansible (2.4).
This issue was not closed, but should have been.

I'm closing it.
